backend/Tasque.DAL/DataContext.cs

```diff
@@ -17,6 +18,7 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         modelBuilder.ApplyConfiguration(new ProjectConfiguration());
         modelBuilder.ApplyConfiguration(new OrganizationConfiguration());
         modelBuilder.ApplyConfiguration(new UserProjectRoleConfiguration());
+        modelBuilder.Seed();
```

Comment:
As i remember, seed is applying every time we make new migration. So, i think, we should comment it for now. Because migration with roles is already created.

Reply:
As far as I know, seeds are added only during the first migration in which they were added. That is, there is no need to comment on them. Or maybe I misunderstood